Guide/Greedy/Stable Marriage/README.md

```diff
@@ -0,0 +1,294 @@
+# Stable Marriage
+
+This problem is introduced in [Dr. Gelfond Applied Algorithms](http://redwood.cs.ttu.edu/~mgelfond/FALL-2012/slides.pdf) and is further discussed in [Stable Marriage Problem Wikipedia](https://en.wikipedia.org/wiki/Stable_marriage_problem)
+
+Category: Greedy
+
+Difficulty: N/A
+
+## Problem
+
+### Overview
+
+Given _n_ men and _n_ women, where each person has ranked all members of the opposite sex in order of preference, marry the men and the women together such that there are no two people of opposite sex who would both rather have each other than their current partners. When there are no such pairs of people, the set of marriages is deemed stable.
+
+_From Wikipedia:_
```

Comment:
Since this is a direct quote, might as well treat it like a quote. See [here](https://guides.github.com/features/mastering-markdown/) for markdown help on GitHub